vformer/attention/window.py

```diff
@@ -57,6 +59,23 @@ def __init__(
         trunc_normal_(self.relative_position_bias_table, std=0.2)
 
     def forward(self, x, mask=None):
+        """
+
+        Parameters
+        ----------
+            x: torch.Tensor
+                input Tensor
+            mask: torch.Tensor
+                Attention mask used for shifted window attention, if None, window attention will be used,
+                else attention mask will be taken into consideration.
+                for more understanding you may refer (this)[https://github.com/microsoft/Swin-Transformer/issues/38]
```

Comment:
For this link to render properly in the documentation, you will have to use the RST format of specifying links.